scenarios/openshift/openshift-apiserver.yml

```diff
@@ -3,5 +3,6 @@
   config:
     namespace_pattern: ^openshift-apiserver$
     label_selector: app=openshift-apiserver-a
+    node_label_selector: node-role.kubernetes.io/master=  # Target master nodes only
```

Comment:
think lets take off the node labels for the base pod scenarios, can keep in the custom app pod one